src/app/app.ts

```diff
@@ -1297,13 +1303,12 @@ function sleepclick() {
     document.getElementById("sleep-img").style.color = '#FFFFFF';
     document.getElementById("water-img").style.color = '#999393';
     document.getElementById("food-img").style.color = '#999393';
-    document.getElementById("title").innerHTML = 'Sleep Tracker <span id="sleeptitle-annotation1">Daily Average</span><span id="sleeptitle-annotation2">Goal</span>';
-    document.getElementById("subtitle").innerHTML = '<span id="sleepsubtitle-annotation1">6.32 hrs</span><span id="sleepsubtitle-annotation2">7.5 hrs</span>'
-    document.getElementById("sleepsubtitle-annotation1").style.color = '#4526A6';
-    document.getElementById("sleepsubtitle-annotation2").style.color = '#4526A6';
-
+    document.getElementById('chart-header-title').innerHTML = '<p class="sleep-ch-title" id="title"> Sleep Tracker </p>'
```

Comment:
Ensure whether do you all the attributes like paragraph tag, class name, ID

Reply:
Ensured the need of tags and class names and modified it